my-first-homework.yaml

```diff
@@ -0,0 +1,44 @@
+dreamTeams:
+  myDreamTeam:
+    lakersTeam:
+      namePlayer: "Luka Doncic"
+      positions: guard
+      number: 77
+      age: 27
+      finalsMvp: false
+
```

Comment:
Sometimes makes sense to have empty lines for readability, although an option without empty lines would be still valid